docs/json/radarr/cf/x264.json

```diff
@@ -1,5 +1,11 @@
 {
   "trash_id": "2899d84dc9372de3408e6d8cc18e9666",
+  "trash_scores": {
```

Comment:
missing default & legacy

Reply:
because they don't have a default/legacy score and the sync apps treat a missing `trash_score` as 0.
we are not "allowed" to use a value of 0 for `trash_score`